packages/entity/src/ComposedSecondaryEntityCache.ts

```diff
@@ -3,7 +3,7 @@ import nullthrows from 'nullthrows';
 import { ISecondaryEntityCache } from './EntitySecondaryCacheLoader';
 
 /**
- * A ISecondaryEntityCache that composes other ISecondaryEntityCache instances.
+ * A ISecondaryEntityCache that composes other ISecondaryEntityCache instances..
  */
 export default class ComposedSecondaryEntityCache<TLoadParams, TFields>
   implements ISecondaryEntityCache<TFields, TLoadParams>
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 26]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->comment outside code<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/expo/entity/245?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->